libstatistics_collector/test/moving_average_statistics/test_moving_average_statistics.cpp

```diff
@@ -57,8 +58,7 @@ class MovingAverageStatisticsTestFixture : public ::testing::Test
   }
 
 protected:
-  std::unique_ptr<moving_average_statistics::MovingAverageStatistics> moving_average_statistics_ =
-    nullptr;
+  std::unique_ptr<MovingAverageStatistics> moving_average_statistics_ = nullptr;
```

Comment:
```suggestion
  std::unique_ptr<MovingAverageStatistics> moving_average_statistics_{nullptr};
```

Reply:
Removed the reference to `nullptr` altogether, since `std::unique_ptr` is default constructed to `nullptr` anyway.